cfgov/unprocessed/css/molecules/full-width-media.less

```diff
@@ -24,10 +24,14 @@
 */
 
 .m-full-width-media {
+    margin-bottom: unit( @grid_gutter-width / @base-font-size-px, em );
```

Comment:
We typically don't add margin to the atomic components.

Reply:
Ah yes, thanks. I noticed that earlier but forgot to go back and fix it.